src/provisioner/provisioner.py

```diff
@@ -32,6 +33,14 @@ class PROVISIONING_STAGES(Enum):
     DEPLOYMENT = "deployment"
     UPGRADE = "upgrade"
 
+class MINIPROVISIONING_STAGES(Enum):
```

Comment:
All class names follow canonical naming

Reply:
done

Reply:
Stages is used for multiple things. Lets use name less conflicting names

```
class ProvisionerStages(Enum):
  DEPLOYMENT
  UPGRADE

class ProvisionerInterfaces(Enum):
  POST_INSTALL
  PREPARE
  CONFIG
  INIT
```